site/docs/reference/connectors-datacatalog/Models/Connection.md

```diff
@@ -1,9 +1,13 @@
 # Connection
-Name of the connection to the data source
+Connection information
```

Comment:
Please expand on which components use the connection information (connectors, modules, storage allocator, optimizer), and that not all components necessarily support all the connection types mentioned.